src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte

```diff
@@ -393,10 +385,6 @@
         &:not(.batch-actions-enabled) {
```

Comment:
Looks like we can still select in this state 😞 

![Screenshot 2023-03-30 at 12 29 56 PM](https://user-images.githubusercontent.com/15069288/228930892-3d370f35-31ba-4888-820a-42c8afc9cd90.png)